### PR TITLE
Extend datetime support through 2099

### LINKED ISSUE #272: riptable date time nano issue with far out dates
It appears that DateTimeNano only goes out as far 2040 EOY. Is this an int64 limitation? Is there any other date time repr in riptable I could use which would avoid this problem?
```
rt.DateTimeNano(['2040-12-30T16:30:00'], from_tz='NYC')
```
yields
```
DateTimeNano(['20401230 16:30:00.000000000'], to_tz='NYC')
```
---
```
rt.DateTimeNano(['2041-12-30T16:30:00'], from_tz='NYC')
```
yields
```
DateTimeNano(['Inv'], to_tz='NYC')
```

### PR DESCRIPTION
**Pull request by OrestZborowski-SIG**
Fixes #272

### CODE REVIEW

**Comment by OrestZborowski-SIG** [comment]
The tests will fail until we deploy riptide_cpp v1.7.5 to get the fix from rtosholdings/riptide_cpp#79

File: riptable/rt_datetime.py [hunk 1]
```diff
@@ -119,9 +119,68 @@
...
+    DAYS_AT_2000 + (94 * DAYS_PER_YEAR) + (24),
+    DAYS_AT_2000 + (95 * DAYS_PER_YEAR) + (24),
+    DAYS_AT_2000 + (96 * DAYS_PER_YEAR) + (24),
+    DAYS_AT_2000 + (97 * DAYS_PER_YEAR) + (25),
+    DAYS_AT_2000 + (98 * DAYS_PER_YEAR) + (25),
+    DAYS_AT_2000 + (99 * DAYS_PER_YEAR) + (25),
 ])
 
-# UTC @ midnight, years 1970 - 2040
+# UTC @ midnight, years 1970 - 2099
```
Comment (jack-pappas) [inline riptable/rt_datetime.py:183]: Could this array just be created by writing:
```python
UTC_1970_SPLITS = UTC_1970_DAY_SPLITS * NANOS_PER_DAY
```

Comment (OrestZborowski-SIG) [reply riptable/rt_datetime.py:183]: Yes, it can!

File: riptable/rt_datetime.py [hunk 2]
```diff
@@ -119,82 +119,69 @@
...
-    NANOS_AT_2000 + (34 * NANOS_PER_YEAR) + (NANOS_PER_DAY * 9),
-    NANOS_AT_2000 + (35 * NANOS_PER_YEAR) + (NANOS_PER_DAY * 9),
-    NANOS_AT_2000 + (36 * NANOS_PER_YEAR) + (NANOS_PER_DAY * 9),
-    NANOS_AT_2000 + (37 * NANOS_PER_YEAR) + (NANOS_PER_DAY * 10),
-    NANOS_AT_2000 + (38 * NANOS_PER_YEAR) + (NANOS_PER_DAY * 10),
-    NANOS_AT_2000 + (39 * NANOS_PER_YEAR) + (NANOS_PER_DAY * 10),
-    NANOS_AT_2000 + (40 * NANOS_PER_YEAR) + (NANOS_PER_DAY * 10),  # 2040
-])
+# UTC @ midnight, years 1970 - 2099
+UTC_1970_SPLITS = UTC_1970_SPLITS = UTC_1970_DAY_SPLITS * NANOS_PER_DAY
```
Comment (jack-pappas) [inline riptable/rt_datetime.py:184]: The variable is being assigned twice here.

Comment (OrestZborowski-SIG) [reply riptable/rt_datetime.py:184]: D'oh!

**Review by jack-pappas — APPROVED** [review]

**Pull request merged**